vertica/datadog_checks/vertica/data/conf.yaml.example

```diff
@@ -158,18 +158,21 @@ instances:
     #
     # client_lib_log_level: <CLIENT_LIB_LOG_LEVEL>
 
-## Log Section (Available for Agent >=6.0)
-##
-## type - mandatory - Type of log input source (tcp / udp / file / windows_event)
-## port / path / channel_path - mandatory - Set port if type is tcp or udp. Set path if type is file. Set channel_path if type is windows_event
-## service - mandatory - Name of the service that generated the log
-## source  - mandatory - Attribute that defines which Integration sent the logs
-## tags: - optional - Add tags to the collected logs
-##
-## Discover Datadog log collection: https://docs.datadoghq.com/logs/log_collection/
-#
-# logs:
-#   - source: vertica
-#     type: file
-#     path: /<CATALOG_PATH>/<DATABASE_NAME>/<NODE_NAME>_catalog/vertica.log
-#     service: vertica
+    ## Log Section
+    ##
+    ## type - required - Type of log input source (tcp / udp / file / windows_event)
+    ## port / path / channel_path - required - Set port if type is tcp or udp.
+    ##                                         Set path if type is file.
+    ##                                         Set channel_path if type is windows_event.
+    ## source  - required - Attribute that defines which Integration sent the logs.
+    ## service - optional - The name of the service that generates the log.
+    ##                      Overrides any `service` defined in the `init_config` section.
+    ## tags - optional - Add tags to the collected logs.
+    ##
+    ## Discover Datadog log collection: https://docs.datadoghq.com/logs/log_collection/
+    #
+    # logs:
+    #   - source: vertica
+    #     type: file
+    #     path: /<CATALOG_PATH>/<DATABASE_NAME>/<NODE_NAME>_catalog/vertica.log
+    #     service: vertica
```

Comment:
```suggestion
```
Let's drop service field here as we're encouraging users to use the service option

Reply:
Removed field